dev/fattest.simplicity/src/componenttest/annotation/TestServlet.java

```diff
@@ -40,6 +38,6 @@
     /**
      * The servlet class to scan for '@Test' annotations, which will be invoked automatically via HTTP GET request
      */
-    Class<? extends HttpServlet> servlet();
+    Class<?> servlet();
```

Comment:
Since we need to remove the compile-time type checks here, perhaps we could add in a runtime type-check on the code that consumes this annotation value to make sure the class is an instance of javax or jakarta HttpServlet?

Reply:
See TestServletProcessor.getTestServletMethods()

Reply:
Done.  See class TestServletProcessor.